Modules/_elementtree.c

```diff
@@ -4334,10 +4258,10 @@ PyInit__elementtree(void)
             expat_capi->MICRO_VERSION != XML_MICRO_VERSION) {
             PyErr_SetString(PyExc_ImportError,
                             "pyexpat version is incompatible");
-            return NULL;
+            goto error;
         }
     } else {
-        return NULL;
+        goto error;
     }
 
     st->str_append = PyUnicode_InternFromString("append");
```

Comment:
Also change this to `goto error`? line-4269

Reply:
Ah, that a result of a bad merge. Thanks!